Merge changes from frontend/homepage

nylla8444 opened the pull request:
Please review manongs @Reiebenezer  @keanedalisay 

Changes:
- added phosphor-icons/react module
- layout config (changed font to Poppins); small additions in globals.css
- new svg assets for hero and footer
- finished 2nd  section for homepage
-  small changes in Header, Footer, and LinkButton jsx

Reiebenezer reviewed (approved):
This part is ok. I have made the same changes as well, with the exception of renaming --accent to --primary and not adding --accent-2.

Reiebenezer reviewed (commented):
For `public/footer.svg`, as mentioned in the gc, it is not preferable to add SVGs to the `public` folder. It is supposed to be under `/src/assets`, like what you did in `hero.svg`. To reference this svg dayon, what is preferable is to import using an import alias and pass it to an `Image` component's src, like this:

```jsx
import bgWave from '@/assets/bg-wave-footer.svg'

export default function Footer() {
    return (
        <Image src={bgWave} className="w-full" />
    );
}
```

An additional note nlg gle is that I already did this on my pull request as well. It's up to @KirkGamo to decide which pull request to keep. 

By the way, `src/assets/hero.svg` is fine.

Reiebenezer commented on src/app/Footer.jsx:
```diff
@@ -1,22 +1,21 @@
     return (
         <footer className="text-white ">
+
             <img src="/footer.svg" alt="" className="w-screen" />
```
Use the `Image` component from next.js here. It's the most optimal one to use.

Reiebenezer commented on src/app/Footer.jsx:
```diff
@@ -25,24 +24,25 @@ export default function Footer() {
                         <div className=" flex flex-col gap-4  ">
                             <a href="" className="flex items-center text-base">
-                                <i className="mail-icon"></i>
+                                <EnvelopeSimple size={40} weight="fill" className="text-[#7ed957] mr-2" />
```
Why are the icons too large? Also, don't pass the size value. Use `className` for the size. Then, use the colors in the color scheme. Don't specify the hex in the icon.

Reiebenezer commented on src/app/Footer.jsx:
```diff
@@ -25,24 +24,25 @@ export default function Footer() {
 
                             <a href="" className="flex items-center text-base">
-                                <i className="fb-icon"></i>
+                                <FacebookLogo size={40} weight="fill" className="text-[#7ed957] mr-2" />
```
Refer to the comment on line 27 for this one.

Reiebenezer commented on src/app/Header.jsx:
```diff
@@ -1,27 +1,42 @@
-                <li><a href="#" className="text-[#1D1D1F] hover:text-gray-400">Services</a></li>
-                <li><a href="#" className="text-[#1D1D1F] hover:text-gray-400">News</a></li>
-                <li><a href="#" className="text-[#1D1D1F] hover:text-gray-400">Contact Us</a></li>
+                <li>
```
Use the colors on the color scheme, again. Don't hardcode the hex color for consistency.

Reiebenezer reviewed (changes requested):
Please check the comments I made. There are a couple requested changes there.

nylla8444 commented:
Please review manongs @Reiebenezer  @keanedalisay 

Changes:
- added heroicons module
- removed phosphorus icons module
- removed hardcoded inline hex values in text colors
- refactored color naming in globals.css
- updated files connected to homepage

Reiebenezer commented on src/assets/arrow-right.svg:
Arrow-right is in heroicons. Don't use an svg on this one.

Reiebenezer commented on src/styles/globals.css:
```diff
@@ -6,14 +6,24 @@
+  --green-300: #8FC867;
+  --green-200: #7ed957;
+  --neutral-900: #1d1d1f;
+
```
Rename these variables to their semantic meaning, like what you added here sa `@theme inline`. Don't use --green-[number], as devs would confuse this color with the preset Tailwind color set.

Reiebenezer reviewed (changes requested):
I added additional comments on specific parts. Please fix

nylla8444 commented:
@Reiebenezer 

Changes done. Kindly review

Reiebenezer reviewed (approved):
Looks good. Okss na ni sya

nylla8444 commented:
all goods manong @Reiebenezer
